Update WebGL conformance test suite 2022-08-02 (48078973d44d6ca6cce1731c505fd2276dede41e)

kkinnunen-apple opened the pull request:
#### a133071172122da396e19c15a7311b7fadfd7545
<pre>
Update WebGL conformance test suite 2022-08-02 (48078973d44d6ca6cce1731c505fd2276dede41e)
<a href="https://bugs.webkit.org/show_bug.cgi?id=243448">https://bugs.webkit.org/show_bug.cgi?id=243448</a>
rdar://problem/97967312

Reviewed by Kenneth Russell.

Update to LayoutTests/webgl/1.0.x, LayoutTests/webgl/2.0.y

* LayoutTests/webgl/1.0.x/conformance/context/context-attribute-preserve-drawing-buffer-antialias-expected.txt: Added.
* LayoutTests/webgl/1.0.x/conformance/context/context-attribute-preserve-drawing-buffer-antialias.html: Added.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-1-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-1-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-2-of-8-expected.txt: Renamed from LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-2-of-8.html: Renamed from LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-3-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-3-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-4-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-4-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-5-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-5-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-6-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-6-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-7-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-7-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-8-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-8-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/misc/hint-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/misc/hint.html: Added.
* LayoutTests/webgl/2.0.y/conformance/context/context-attribute-preserve-drawing-buffer-antialias-expected.txt: Added.
* LayoutTests/webgl/2.0.y/conformance/context/context-attribute-preserve-drawing-buffer-antialias.html: Added.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-1-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-1-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-2-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-2-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-3-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-3-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-4-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-4-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-5-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-5-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-6-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-6-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-7-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-7-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-8-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-8-of-8.html: Renamed from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/misc/hint-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/misc/hint.html: Added.
* LayoutTests/webgl/2.0.y/conformance2/attribs/invalid-vertex-attribs-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance2/attribs/invalid-vertex-attribs.html: Added.
* LayoutTests/webgl/2.0.y/conformance2/renderbuffers/multisample-draws-between-blits-expected.txt: Added.
* LayoutTests/webgl/2.0.y/conformance2/renderbuffers/multisample-draws-between-blits.html: Added.
* LayoutTests/webgl/2.0.y/conformance2/rendering/framebuffer-to-texture-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance2/rendering/framebuffer-to-texture.html: Added.
* LayoutTests/webgl/2.0.y/conformance2/rendering/multisampling-depth-resolve-expected.txt: Added.
* LayoutTests/webgl/2.0.y/conformance2/rendering/multisampling-depth-resolve.html: Added.
* LayoutTests/webgl/2.0.y/conformance2/rendering/vertex-id-large-count-expected.txt: Renamed from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance2/rendering/vertex-id-large-count.html: Added.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_matrix_combine-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_matrix_combine.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_matrix_to_matrix-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_matrix_to_matrix.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_scalar_to_matrix-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_scalar_to_matrix.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_scalar_to_scalar-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_scalar_to_scalar.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_scalar_to_vector-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_scalar_to_vector.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_combine-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_combine.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_illegal-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_illegal.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_to_scalar-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_to_scalar.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_to_vector-expected.txt: Renamed from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_to_vector.html: Renamed from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/js-test-pre.js:
(debug):
(testPassed):
* LayoutTests/webgl/resources/webgl_test_files/conformance/attribs/gl-enable-vertex-attrib.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/canvas/buffer-preserve-test.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/context/constants-and-properties.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/context/context-attribute-preserve-drawing-buffer-antialias.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/context/context-lost.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/extensions/angle-instanced-arrays.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/extensions/ext-texture-compression-bptc.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/extensions/oes-vertex-array-object.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/extensions/webgl-draw-buffers.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/extensions/webgl-multi-draw.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/extensions/webgl-webcodecs-video-frame.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-1-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-2-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-3-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-4-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-5-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-6-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-7-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-8-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words.html: Removed.
* LayoutTests/webgl/resources/webgl_test_files/conformance/misc/hint.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/programs/program-handling.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/rendering/blending.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/rendering/color-mask-preserved-during-implicit-clears.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/rendering/draw-webgl-to-canvas-2d-repeatedly.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/rendering/gl-scissor-fbo-test.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/textures/misc/texture-npot-video.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/textures/misc/texture-video-transparent.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/attribs/invalid-vertex-attribs.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance2/buffers/get-buffer-sub-data-validity.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/canvas/compositing.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance2/context/constants-and-properties-2.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/context/incorrect-context-object-behaviour.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/extensions/ext-disjoint-timer-query-webgl2.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/extensions/ext-texture-norm16.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/extensions/oes-draw-buffers-indexed.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/renderbuffers/framebuffer-object-attachment.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/renderbuffers/invalidate-framebuffer.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/renderbuffers/multisample-draws-between-blits.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance2/rendering/clearbuffer-and-draw.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/rendering/framebuffer-to-texture.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance2/rendering/multisampling-depth-resolve.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance2/rendering/vertex-id-large-count.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance2/textures/misc/copy-texture-image.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/textures/misc/tex-image-with-bad-args.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/transform_feedback/transform_feedback.html:
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.template: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_matrix_combine.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_matrix_combine.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_matrix_to_matrix.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_matrix_to_matrix.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_scalar_to_matrix.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_scalar_to_matrix.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_scalar_to_scalar.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_scalar_to_scalar.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_scalar_to_vector.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_scalar_to_vector.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_combine.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_combine.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_illegal.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_illegal.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_to_scalar.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_to_scalar.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_to_vector.html: Renamed from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_to_vector.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/split-conversions.js: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/functional/gles3/es3fNegativeTextureApiTests.js:
(goog.scope.es3fNegativeTextureApiTests.init):
* LayoutTests/webgl/resources/webgl_test_files/js/js-test-post.js:
* LayoutTests/webgl/resources/webgl_test_files/js/tests/compositing-test.js: Added.
(waitForComposite):
(async testPreserveDrawingBufferFalse):
(async testPreserveDrawingBufferTrue):
(setupWebGL):
* LayoutTests/webgl/resources/webgl_test_files/js/tests/ext-texture-filter-anisotropic.js:
(runHintTestDisabled):
(runSamplerTestDisabled):
* LayoutTests/webgl/resources/webgl_test_files/js/tests/invalid-vertex-attrib-test.js: Added.
(async testPreserveDrawingBufferTrue):
(setupWebGL):
(createInvalidAttribTestFn.return.async invalidAttribTestFn):
(createInvalidAttribTestFn):
* LayoutTests/webgl/resources/webgl_test_files/js/tests/shader-with-non-reserved-words.js: Added.
(testWord):
(async testNonReservedWords):
* LayoutTests/webgl/resources/webgl_test_files/js/tests/tex-image-and-sub-image-2d-with-video.js:
(init):
(runOneIteration):
(runTest.runTexImageTest.):
(runTest.runTexImageTest):
(runTest):
(generateTest):
* LayoutTests/webgl/resources/webgl_test_files/js/tests/tex-image-and-sub-image-utils.js:
(unpackColorSpacesToTest):
(testCaseWithUnpackColorSpace):
(crossProductTestCasesWithUnpackColorSpaces):
(formatTolerance):
(tolerance):
(TexImageUtils):
* LayoutTests/webgl/resources/webgl_test_files/js/tests/tex-input-validation.js:
* LayoutTests/webgl/resources/webgl_test_files/js/webgl-test-utils.js:
(create3DContext):
(namedColorInColorSpace):
(colorAsSampledWithInternalFormat):
(WebGLTestUtils):

Canonical link: <a href="https://commits.webkit.org/253107@main">https://commits.webkit.org/253107@main</a>
</pre>

kenrussell reviewed (approved):
Great! Looks good to me.

@lexaknyazev FYI.

kkinnunen-apple committed:
Update WebGL conformance test suite 2022-08-02 (48078973d44d6ca6cce1731c505fd2276dede41e)
https://bugs.webkit.org/show_bug.cgi?id=243448
rdar://problem/97967312

Reviewed by Kenneth Russell.

Update to LayoutTests/webgl/1.0.x, LayoutTests/webgl/2.0.y

* LayoutTests/webgl/1.0.x/conformance/context/context-attribute-preserve-drawing-buffer-antialias-expected.txt: Added.
* LayoutTests/webgl/1.0.x/conformance/context/context-attribute-preserve-drawing-buffer-antialias.html: Added.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-1-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-1-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-2-of-8-expected.txt: Renamed from LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-2-of-8.html: Renamed from LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-3-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-3-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-4-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-4-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-5-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-5-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-6-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-6-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-7-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-7-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-8-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-8-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/1.0.x/conformance/misc/hint-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/1.0.x/conformance/misc/hint.html: Added.
* LayoutTests/webgl/2.0.y/conformance/context/context-attribute-preserve-drawing-buffer-antialias-expected.txt: Added.
* LayoutTests/webgl/2.0.y/conformance/context/context-attribute-preserve-drawing-buffer-antialias.html: Added.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-1-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-1-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-2-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-2-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-3-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-3-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-4-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-4-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-5-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-5-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-6-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-6-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-7-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-7-of-8.html: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-8-of-8-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/glsl/misc/shader-with-non-reserved-words-8-of-8.html: Renamed from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words.html.
* LayoutTests/webgl/2.0.y/conformance/misc/hint-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance/misc/hint.html: Added.
* LayoutTests/webgl/2.0.y/conformance2/attribs/invalid-vertex-attribs-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance2/attribs/invalid-vertex-attribs.html: Added.
* LayoutTests/webgl/2.0.y/conformance2/renderbuffers/multisample-draws-between-blits-expected.txt: Added.
* LayoutTests/webgl/2.0.y/conformance2/renderbuffers/multisample-draws-between-blits.html: Added.
* LayoutTests/webgl/2.0.y/conformance2/rendering/framebuffer-to-texture-expected.txt: Copied from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance2/rendering/framebuffer-to-texture.html: Added.
* LayoutTests/webgl/2.0.y/conformance2/rendering/multisampling-depth-resolve-expected.txt: Added.
* LayoutTests/webgl/2.0.y/conformance2/rendering/multisampling-depth-resolve.html: Added.
* LayoutTests/webgl/2.0.y/conformance2/rendering/vertex-id-large-count-expected.txt: Renamed from LayoutTests/webgl/1.0.x/conformance/glsl/misc/shader-with-non-reserved-words-expected.txt.
* LayoutTests/webgl/2.0.y/conformance2/rendering/vertex-id-large-count.html: Added.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_matrix_combine-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_matrix_combine.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_matrix_to_matrix-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_matrix_to_matrix.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_scalar_to_matrix-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_scalar_to_matrix.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_scalar_to_scalar-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_scalar_to_scalar.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_scalar_to_vector-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_scalar_to_vector.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_combine-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_combine.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_illegal-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_illegal.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_to_scalar-expected.txt: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_to_scalar.html: Copied from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_to_vector-expected.txt: Renamed from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions-expected.txt.
* LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions_vector_to_vector.html: Renamed from LayoutTests/webgl/2.0.y/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/js-test-pre.js:
(debug):
(testPassed):
* LayoutTests/webgl/resources/webgl_test_files/conformance/attribs/gl-enable-vertex-attrib.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/canvas/buffer-preserve-test.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/context/constants-and-properties.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/context/context-attribute-preserve-drawing-buffer-antialias.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/context/context-lost.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/extensions/angle-instanced-arrays.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/extensions/ext-texture-compression-bptc.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/extensions/oes-vertex-array-object.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/extensions/webgl-draw-buffers.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/extensions/webgl-multi-draw.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/extensions/webgl-webcodecs-video-frame.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-1-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-2-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-3-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-4-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-5-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-6-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-7-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words-8-of-8.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/glsl/misc/shader-with-non-reserved-words.html: Removed.
* LayoutTests/webgl/resources/webgl_test_files/conformance/misc/hint.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance/programs/program-handling.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/rendering/blending.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/rendering/color-mask-preserved-during-implicit-clears.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/rendering/draw-webgl-to-canvas-2d-repeatedly.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/rendering/gl-scissor-fbo-test.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/textures/misc/texture-npot-video.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance/textures/misc/texture-video-transparent.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/attribs/invalid-vertex-attribs.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance2/buffers/get-buffer-sub-data-validity.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/canvas/compositing.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance2/context/constants-and-properties-2.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/context/incorrect-context-object-behaviour.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/extensions/ext-disjoint-timer-query-webgl2.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/extensions/ext-texture-norm16.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/extensions/oes-draw-buffers-indexed.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/renderbuffers/framebuffer-object-attachment.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/renderbuffers/invalidate-framebuffer.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/renderbuffers/multisample-draws-between-blits.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance2/rendering/clearbuffer-and-draw.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/rendering/framebuffer-to-texture.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance2/rendering/multisampling-depth-resolve.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance2/rendering/vertex-id-large-count.html: Added.
* LayoutTests/webgl/resources/webgl_test_files/conformance2/textures/misc/copy-texture-image.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/textures/misc/tex-image-with-bad-args.html:
* LayoutTests/webgl/resources/webgl_test_files/conformance2/transform_feedback/transform_feedback.html:
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.template: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_matrix_combine.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_matrix_combine.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_matrix_to_matrix.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_matrix_to_matrix.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_scalar_to_matrix.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_scalar_to_matrix.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_scalar_to_scalar.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_scalar_to_scalar.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_scalar_to_vector.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_scalar_to_vector.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_combine.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_combine.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_illegal.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_illegal.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_to_scalar.html: Copied from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_to_scalar.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_to_vector.html: Renamed from LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions.html.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/conversions_vector_to_vector.test: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/data/gles3/shaders/split-conversions.js: Added.
* LayoutTests/webgl/resources/webgl_test_files/deqp/functional/gles3/es3fNegativeTextureApiTests.js:
(goog.scope.es3fNegativeTextureApiTests.init):
* LayoutTests/webgl/resources/webgl_test_files/js/js-test-post.js:
* LayoutTests/webgl/resources/webgl_test_files/js/tests/compositing-test.js: Added.
(waitForComposite):
(async testPreserveDrawingBufferFalse):
(async testPreserveDrawingBufferTrue):
(setupWebGL):
* LayoutTests/webgl/resources/webgl_test_files/js/tests/ext-texture-filter-anisotropic.js:
(runHintTestDisabled):
(runSamplerTestDisabled):
* LayoutTests/webgl/resources/webgl_test_files/js/tests/invalid-vertex-attrib-test.js: Added.
(async testPreserveDrawingBufferTrue):
(setupWebGL):
(createInvalidAttribTestFn.return.async invalidAttribTestFn):
(createInvalidAttribTestFn):
* LayoutTests/webgl/resources/webgl_test_files/js/tests/shader-with-non-reserved-words.js: Added.
(testWord):
(async testNonReservedWords):
* LayoutTests/webgl/resources/webgl_test_files/js/tests/tex-image-and-sub-image-2d-with-video.js:
(init):
(runOneIteration):
(runTest.runTexImageTest.):
(runTest.runTexImageTest):
(runTest):
(generateTest):
* LayoutTests/webgl/resources/webgl_test_files/js/tests/tex-image-and-sub-image-utils.js:
(unpackColorSpacesToTest):
(testCaseWithUnpackColorSpace):
(crossProductTestCasesWithUnpackColorSpaces):
(formatTolerance):
(tolerance):
(TexImageUtils):
* LayoutTests/webgl/resources/webgl_test_files/js/tests/tex-input-validation.js:
* LayoutTests/webgl/resources/webgl_test_files/js/webgl-test-utils.js:
(create3DContext):
(namedColorInColorSpace):
(colorAsSampledWithInternalFormat):
(WebGLTestUtils):

Canonical link: https://commits.webkit.org/253107@main
